config/src/main/scala/bloop/config/Config.scala

```diff
@@ -59,6 +59,46 @@ object Config {
     private[bloop] val empty: Scala = Scala("", "", "", Array(), Array())
   }
 
+  sealed abstract class Platform(val name: String)
+  object Platform {
+    private[bloop] val default: Platform = JVM
+
+    case object JS extends Platform("js")
+    case object JVM extends Platform("jvm")
+    case object Native extends Platform("native")
+
+    def apply(platform: String): Platform = platform.toLowerCase match {
+      case JS.name => JS
+      case JVM.name => JVM
+      case Native.name => Native
+      case _ => throw new IllegalArgumentException(s"Unknown platform: '$platform'")
+    }
+  }
+
+  /**
+   * Configures how to start and use the Scala Native toolchain, if needed.
+   * See http://javadoc.io/doc/org.scala-native/tools_2.10/0.3.7 for the description of the fields.
```

Comment:
This link points to the root of the scaladoc, would it be possible to link to the class containing all the fields? :smile:

Reply:
lol